contrib/hashicorp/consul/example_test.go

```diff
@@ -0,0 +1,46 @@
+package consul
+
+import (
+	"context"
+	"fmt"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	consul "github.com/hashicorp/consul/api"
+)
+
+func Example() {
+	// Get a new Consul client
+	client, err := NewClient(consul.DefaultConfig(), WithServiceName("consul.example"))
+	if err != nil {
+		panic(err)
```

Comment:
Forgot: let's remove _all_ `panic` calls. Replace them with `log.Fatal` here and with `t.Fatal` in tests please.

Reply:
Done